Back End/MemorizeWords/MemorizeWords/Infrastructure/Persistance/Repository/WordAnswerRepository.cs

```diff
@@ -93,10 +94,27 @@ private void ValidationAnswer(WordAnswerRequest wordAnswerRequest)
         private bool GetGivenAnswer(WordAnswerRequest wordAnswerRequest, out WordEntity wordEntity)
         {
             wordEntity = _dbContext.Word.FirstOrDefault(x => x.Id == wordAnswerRequest.WordId) ?? throw new KeyNotFoundBusinessException($"wordId: {wordAnswerRequest.WordId} couldn't found");
+            
+            wordEntity = GetCrossCheckWord(wordAnswerRequest.AnswerLanguageType, wordEntity);
 
             bool answer = wordEntity.Meaning.ToUpperInvariant().Equals(wordAnswerRequest.GivenAnswerMeaning.ToUpperInvariant().ToUpper(), StringComparison.OrdinalIgnoreCase);
             return answer;
         }
+
+        private WordEntity GetCrossCheckWord(int answerLanguageType, WordEntity wordEntity)
+        {
+            if ((LanguageType)answerLanguageType == LanguageType.LearningLanguage)
+            {
+                wordEntity.Meaning = wordEntity.Word;
```

Comment:
Dont change entities ever if you don't want to change the values from DB. Please return Word property as string here

Reply:
Done